scripts/install_third_party_libs.py

```diff
@@ -0,0 +1,271 @@
+            # The Skulpt setup function needs to be tweaked. It fails without
+            # certain third party commands. These are only used for unit tests
+            # and generating documentation and are not necessary when building
+            # Skulpt.
+            for line in fileinput.input(
+                    files=[skulpt_filepath], inplace=True):
+                # Inside this loop the STDOUT will be redirected to the file,
+                # skulpt.py. The end='' is needed to avoid double line breaks.
+                python_utils.PRINT(
+                    line.replace('ret = test()', 'ret = 0'),
+                    end='')
+
+            for line in fileinput.input(
+                    files=[skulpt_filepath], inplace=True):
+                # Inside this loop the STDOUT will be redirected to the file,
+                # skulpt.py. The end='' is needed to avoid double line breaks.
+                python_utils.PRINT(
+                    line.replace('  doc()', '  pass#doc()'),
+                    end='')
+
+            for line in fileinput.input(
+                    files=[skulpt_filepath], inplace=True):
+                # This and the next command disable unit and compressed unit
+                # tests for the compressed distribution of Skulpt. These
+                # tests don't work on some Ubuntu environments and cause a
+                # libreadline dependency issue.
+                python_utils.PRINT(
+                    line.replace(
+                        'ret = os.system(\'{0}',
+                        'ret = 0 #os.system(\'{0}'),
+                    end='')
+
+            for line in fileinput.input(
+                    files=[skulpt_filepath], inplace=True):
+                python_utils.PRINT(
+                    line.replace('ret = rununits(opt=True)', 'ret = 0'),
+                    end='')
+
+            subprocess.call(['python', skulpt_filepath, 'dist'])
+
+            # Return to the Oppia root folder.
+            os.chdir(common.CURR_DIR)
+
+        # Move the build directory to the static resources folder.
+        shutil.copytree(
+            os.path.join(
+                common.OPPIA_TOOLS_DIR, 'skulpt-0.10.0/skulpt/dist/'),
+            os.path.join(common.THIRD_PARTY_DIR, 'static/skulpt-0.10.0'))
+
+
+def maybe_install_dependencies(
+        skip_installing_third_party_libs, run_minified_tests):
+    """Parse additional command line arguments."""
+    if skip_installing_third_party_libs is False:
+        # Install third party dependencies.
+        main()
+        # Ensure that generated JS and CSS files are in place before running the
+        # tests.
+        python_utils.PRINT('Running build task with concatenation only')
+        build.main(argv=[])
+
+    if run_minified_tests is True:
+        python_utils.PRINT(
+            'Running build task with concatenation and minification')
+        build.main(argv=['--prod_env'])
```

Comment:
Are you sure we should always run in prod mode?

Reply:
That's how it is in the original script.